disperser/cmd/controller/flags/flags.go

```diff
@@ -203,6 +203,12 @@ var (
 		Required: false,
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "DISPERSER_KMS_KEY_ID"),
 	}
+	DisperserPrivateKeyFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "disperser-private-key"),
+		Usage:    "Private key for signing disperser requests (hex format without 0x prefix, alternative to KMS)",
```

Comment:
would be good to update the DisperserKMSKeyIDFlag as well to mention that its optional and mutually exclusive with this PrivateKeyFlag